[dagstermill] Guide refresh (#10539)
dagster-io/dagster: docs/content/integrations/dagstermill/using-notebooks-with-dagster.mdx

   <Image
   src="/images/integrations/dagstermill/dagit-three.png"
   width={2870}
   height={1398}
   />

3. Click the **Materialize all** button near the top right corner of the page, which will launch a run to materialize the assets.

That's it! You now have working Jupyter and Dagster assets!

---

## Extra credit: Fetch a Dagster asset in a Jupyter notebook

What if you want to do additional analysis of the Iris dataset and create a new notebook? How can you accomplish this without duplicating code or re-fetching data?

The answer is simple: use the `iris_dataset` Dagster asset!

In the Jupyter notebook, import the Dagster repository and use the <PyObject object="RepositoryDefinition" method="load_asset_value" /> function to load the data for the `iris_dataset` asset we created in [Step 5.1: Create the Iris dataset asset](#step-51-create-the-iris-dataset-asset):

```python
from tutorial_template import template_tutorial

iris = template_tutorial.load_asset_value("iris_dataset")
```

Then, whenever you run the notebook using Jupyter, you'll be able to work with the `iris_dataset` asset:

```shell
jupyter notebook /path/to/new/notebook.ipynb
```

> Behind the scenes, when `load_asset_value` is called, Dagster fetches the value of `iris_dataset` that was most recently materialized and stored by an I/O manager.

Author: @yuhan hows this? should there be more explanation?

Reviewer: lg2m

---
title: Using Jupyter Notebooks with Papermill and Dagster Tutorial
description: The Dagstermill package lets you run notebooks using the Dagster tools and integrate them into your data pipelines.
---

> # Using Jupyter Notebooks with Papermill and Dagster

Reviewer 2: This just popped up in my notifications - sorry!

I don't think notebook is a proper noun, at least the guide doesn't use it as such. Headings and titles should use sentence case, so this should be `notebook`

Author: oops! on it

)
```

> Next, we do some descriptive analysis to explore the dataset. If you execute these cells, several plots of the Iris dataset will be created:

Reviewer 2: ```suggestion
Next, we'll perform some descriptive analysis to explore the dataset. If you execute these cells, several plots of the Iris dataset will be created:
```
